### PR TITLE
Update contracts to alpha 15

### PR DESCRIPTION
**Pull request by anxolin**
#338 

Service PR: https://github.com/gnosis/gp-v2-services/pull/413

## Changes
* Updates contract to Alpha 15
* Sets the receiver, either to the selected receiver, or to 0x0
* Makes sure the AppId is a 32 bytes number. 

## Additionally
* Also adds env vars to test the API locally.

## Notes for the reviewer
* Declare a enum that is meant to be deleted. Check out the TODO. A future NPM package will expose the enum so we can delete that code.

## Test
To test with a local API, once the API is up and listening you can setup these envs in a file `.env.local`

```ts
# API
REACT_APP_API_BASE_URL_MAINNET=http://localhost:8080/api/v1
REACT_APP_API_BASE_URL_RINKEBY=http://localhost:8080/api/v1
REACT_APP_API_BASE_URL_XDAI=http://localhost:8080/api/v1
```

You will need to restart the app, if it was already running.

### CODE REVIEW

**Comment by vkgnosis** [comment]
I think the problem is `"signignScheme":0` which should be  `enum: [eip712, ethsign]` so in this case probably `"eip712"`.

File: src/custom/utils/signatures.ts [hunk 1]
```diff
@@ -52,6 +52,33 @@ export enum SigningScheme {
 }
 export type EcdsaSigningScheme = SigningScheme.EIP712 | SigningScheme.ETHSIGN
 
+interface SchemaInfo {
+  libraryValue: number
+  apiValue: string
+}
+const mapSigningSchema: Map<SigningScheme, SchemaInfo> = new Map([
+  [SigningScheme.EIP712, { libraryValue: 0, apiValue: 'eip712' }],
+  [SigningScheme.ETHSIGN, { libraryValue: 0, apiValue: 'ethsign' }]
```
Comment (vkgnosis) [inline src/custom/utils/signatures.ts:61]: copy paste error?
```suggestion
  [SigningScheme.ETHSIGN, { libraryValue: 1, apiValue: 'ethsign' }]
```

Comment (anxolin) [reply src/custom/utils/signatures.ts:61]: yup! good catch!

**Review by vkgnosis — APPROVED** [review]

File: .env.production [hunk 2]
```diff
@@ -15,3 +15,11 @@ REACT_APP_DOMAIN_PREFIX_PROD=cow.trade
...
 REACT_APP_GOOGLE_ANALYTICS_ID_STAGING="UA-190948266-4"
 REACT_APP_GOOGLE_ANALYTICS_ID_PROD="UA-190948266-1"
+REACT_APP_PORTIS_ID="c0e2bf01-4b08-4fd5-ac7b-8e26b58cd236"
+REACT_APP_FORTMATIC_KEY="pk_live_F937DF033A1666BF"
+REACT_APP_GOOGLE_ANALYTICS_ID="UA-190948266-1"
+
+# API
+#REACT_APP_API_BASE_URL_MAINNET=https://protocol-mainnet.dev.gnosisdev.com/api/v1
+#REACT_APP_API_BASE_URL_RINKEBY=https://protocol-rinkeby.dev.gnosisdev.com/api/v1
+#REACT_APP_API_BASE_URL_XDAI=https://protocol-xdai.dev.gnosisdev.com/api/v1
```
Comment (alfetopito) [inline .env.production:25]: Are you going to leave these commented out?

**Review by alfetopito — APPROVED** [review]

File: .env.production (hunk 2 above)
Comment (anxolin) [reply .env.production:25]: yep, is just an example

**Pull request merged**